### PR TITLE
Add cloud account to Cluster

### PR DESCRIPTION
**Pull request by hakonhall**

### CODE REVIEW

File: node-repository/src/main/java/com/yahoo/vespa/hosted/provision/applications/Cluster.java [hunk 1]
```diff
@@ -58,6 +61,7 @@ public Cluster(ClusterSpec.Id id,
         this.max = Objects.requireNonNull(maxResources);
         this.groupSize = Objects.requireNonNull(groupSize);
         this.required = required;
+        this.cloudAccount = cloudAccount;
```
Comment (mpolden) [inline node-repository/src/main/java/com/yahoo/vespa/hosted/provision/applications/Cluster.java:64]: Nit: Missing `Objects.requireNonNull`

File: node-repository/src/main/java/com/yahoo/vespa/hosted/provision/persistence/ApplicationSerializer.java [hunk 2]
```diff
@@ -56,6 +57,7 @@ public class ApplicationSerializer {
     private static final String maxResourcesKey = "max";
     private static final String groupSizeKey = "groupSize";
     private static final String requiredKey = "required";
+    private static final String cloudAccountKey = "cloud-account";
```
Comment (mpolden) [inline node-repository/src/main/java/com/yahoo/vespa/hosted/provision/persistence/ApplicationSerializer.java:60]: Nit: Make it camel-case like the others?

**Review by mpolden — DISMISSED** [review]
👍

File: node-repository/src/main/java/com/yahoo/vespa/hosted/provision/applications/Cluster.java (hunk 1 above)
Comment (hakonhall) [reply node-repository/src/main/java/com/yahoo/vespa/hosted/provision/applications/Cluster.java]: Fixed

File: node-repository/src/main/java/com/yahoo/vespa/hosted/provision/persistence/ApplicationSerializer.java (hunk 2 above)
Comment (hakonhall) [reply node-repository/src/main/java/com/yahoo/vespa/hosted/provision/persistence/ApplicationSerializer.java]: Fixed

**Review by mpolden — APPROVED** [review]

**Pull request merged**